rdtools/availability.py

```diff
@@ -0,0 +1,620 @@
+"""
+Functions for detecting and quantifying production loss from photovoltaic
+system downtime events.
+
+The availability module is currently experimental. The API, results,
+and default behaviors may change in future releases (including MINOR
+and PATCH releases) as the code matures.
+"""
+
+import rdtools
+
+import pandas as pd
+import numpy as np
+from scipy.interpolate import interp1d
+import warnings
+
+warnings.warn(
+    'The availability module is currently experimental. The API, results, '
+    'and default behaviors may change in future releases (including MINOR '
+    'and PATCH releases) as the code matures.'
+)
```

Comment:
Wow this is a loud warning!

Reply:
The semantic versioning docs are pretty loud themselves. Must be an echo.  ;-)
https://semver.org/